NtupleTools/python/templates/electrons.py

```diff
@@ -106,6 +106,16 @@
     objectNearestZMass = 'closestZElectron({object_idx},"")',
     # lowest invariant mass
     objectLowestMll = 'smallestMee({object_idx},"")',
+
+
+
+    objectGenMotherPt =  '? (getDaughterGenParticleMotherSmart({object_idx}, 11, 0).isAvailable && getDaughterGenParticleMotherSmart({object_idx}, 11, 0).isNonnull) ? getDaughterGenParticleMotherSmart({object_idx}, 11, 0).pt() : -999', 
```

Comment:
Just an organizational thing: can you move this up to https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/NtupleTools/python/templates/electrons.py#L92 so its with the rest of the gen stuff?

And also add it to muons here: https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/NtupleTools/python/templates/muons.py#L57